bokeh/plotting.py

```diff
@@ -410,13 +410,13 @@ def output_file(filename, title="Bokeh Plot", autosave=False, mode="inline", roo
         print("Session output file '%s' already exists, will be overwritten." % filename)
 
 
-def show(obj=None, browser=None, new="tab", url=None):
-    """ 'shows' a plot object or the current plot, by auto-raising the window or tab
-    displaying the current plot (for file/server output modes) or displaying
-    it in an output cell (IPython notebook).
+def show(obj, browser=None, new="tab", url=None):
+    """ 'shows' a plot object, by auto-raising the window or tab displaying the
+    current plot (for file/server output modes) or displaying it in an output
+    cell (IPython/Jupyter notebook).
 
     Args:
-        obj (Widget/Plot object, optional): it accepts a plot object and just shows it.
+        obj (Widget/Plot object): requires a plot object and just shows it.
```

Comment:
Thanks for the detailed explanation... in Spanish is also the "objeto directo" (direct obeject) but we call "all the thing" after the verb a "predicado" (predicate)... thanks again to take some of your time on this...